README.md

```diff
@@ -120,6 +120,12 @@ export EXTERNAL_SAT_ORG='Sat6-CI'
 export EXTERNAL_SAT_ACTIVATION_KEY='Satellite QA RHEL7'
 ```
 
+To use beta repositories during upgrade, set the following environment variable
+
+```
+export FOREMAN_MAINTAIN_USE_BETA='ON'
```

Comment:
@kgaikwad, I wonder if it wouldn't be more convenient/consistent to use `export FOREMAN_MAINTAIN_USE_BETA=1` as in other CLI tools we use 1/true/yes for positive values. What do you think? Otherwise it looks good to go.

Reply:
@mbacovsky,  done :+1: